pkg/astrolabe/protected_entity_type_manager.go

```diff
@@ -29,6 +29,30 @@ const (
 	AllocateObjectWithID CopyCreateOptions = 3
 )
 
+func StringFromCopyCreateOption(options CopyCreateOptions) string {
```

Comment:
Can we use enum type instead?

What's the thinking behind using string parsing here?

Reply:
This was to handle sending things over HTTP and GRPC